### PR TITLE
Update instruction/settings to support authapp's new dependency

### PR DESCRIPTION
**Pull request by rpdome**
With a help from Lord Tito from Authenticator app team.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,23 +8,38 @@ This repository contains a build gradle and git alias commands for building ADAL
...
+- [Authenticator App Project](https://msazure.visualstudio.com/One/_git/AD-MFA-phonefactor-phoneApp-android)
 1. Go to //myaccess
 2. Send a request to join "Identity Apps Team - 18174"
 3. Install [Git Credential Manager for Windows](https://github.com/Microsoft/Git-Credential-Manager-for-Windows) or [Git Credential Manager for Mac and Linux](https://github.com/Microsoft/Git-Credential-Manager-for-Mac-and-Linux) and setup with your MSFT credential.
 
+- [Private GitHub Repository](https://repos.opensource.microsoft.com/)
+1. Go to https://repos.opensource.microsoft.com/
+2. Join 'AzureAD' organization (to get an access to Broker)
+3. Join 'Microsoft' organization (to get an access to Authenticator app's submodule.)
+
```
Comment (rpdome) [inline README.md:36]: nit: also add https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh

File: README.md [hunk 2]
```diff
@@ -8,23 +8,41 @@ This repository contains a build gradle and git alias commands for building ADAL
...
+- [Authenticator App Project](https://msazure.visualstudio.com/One/_git/AD-MFA-phonefactor-phoneApp-android)
 1. Go to //myaccess
 2. Send a request to join "Identity Apps Team - 18174"
 3. Install [Git Credential Manager for Windows](https://github.com/Microsoft/Git-Credential-Manager-for-Windows) or [Git Credential Manager for Mac and Linux](https://github.com/Microsoft/Git-Credential-Manager-for-Mac-and-Linux) and setup with your MSFT credential.
 
+- [Private GitHub Repositories](https://repos.opensource.microsoft.com/)
+1. Go to https://repos.opensource.microsoft.com/. You'll need a github account.
+2. Join 'AzureAD' organization (to get an access to Broker)
+3. Join 'Microsoft' organization (to get an access to Authenticator app's submodule.)
```
Comment (AdamBJohnsonx) [inline README.md:35]:
```suggestion
2. Join 'AzureAD' organization (to get an access to Broker) via https://repos.opensource.microsoft.com/AzureAD/join
3. Join 'Microsoft' organization (to get an access to Authenticator app's submodule.) via https://repos.opensource.microsoft.com/Microsoft/join
```

Comment (rpdome) [reply README.md]: I already joined... so I couldn't get the link 😅

Comment (AdamBJohnsonx) [reply README.md]: I poked around in order to find out the site structure and then tested it on a couple of different repos. ;-)

**Pull request merged**